website/docs/streaming-lakehouse/integrate-data-lakes/iceberg.md

```diff
@@ -307,29 +466,81 @@ When integrating with Iceberg, Fluss automatically converts between Fluss data t
 
 ## Maintenance and Optimization
 
-### Auto Compaction
+### 🔧 Auto Compaction
```

Comment:
The icon is duplicated between the `Basic Configuration` title and the `Reading with Other Engines` title.